ui-v2/app/utils/http/consul.js

```diff
@@ -0,0 +1,3 @@
+export const HEADERS_SYMBOL = '__consul_ui_http_headers__';
```

Comment:
I wasn't quite sure where this should live at first, this seems to make most sense though, it's http 'related' and specific to `consul`